src/test/groovy/graphql/ScalarsQuerySchema.java

```diff
@@ -18,18 +18,6 @@ public class ScalarsQuerySchema {
 
     public static final GraphQLObjectType queryType = newObject()
             .name("QueryType")
-            // Static scalars
```

Comment:
This was added a long time ago, before the GraphQL Float spec was updated to specifically not allow NaN and positive/negative infinity values